src/modals/send-modal/send-modal.tsx

```diff
@@ -36,6 +45,9 @@ export const SendModal: FC = () => {
   const { asset: initialAsset } = useRoute<RouteProp<ModalsParamList, ModalsEnum.Send>>().params;
   const { goBack } = useNavigation();
 
+  const { publicKeyHash: senderPublicKeyHash } = useSelectedAccountSelector();
```

Comment:
remove this destructuring please, just use `selectedAccount. publicKeyHash` instead